segmentation/utils/spatialanalysis_utils_test.py

```diff
@@ -10,3 +13,78 @@ def test_calc_dist_matrix():
     dist_matrix = spatialanalysis_utils.calc_dist_matrix(test_mat)
     real_mat = np.array([[0, 5], [5, 0]])
     assert np.array_equal(dist_matrix, real_mat)
+
+
+def test_distmat():
```

Comment:
This is going to fail because these files aren't available on github. Keep it here, but comment it out. When we come back in a month and refactor it, we may want to have this code here to double check it runs

Reply:
Also, all the test functions should be named like the one above, with _test at the end of the function that's being tested. You can just put this inside the test function above